armi/materials/material.py

```diff
@@ -400,43 +400,44 @@ def volumetricExpansion(self, Tk=None, Tc=None):
         pass
 
     def getTemperatureAtDensity(
-        self, targetDensity: float, temperatureGuessInC: float
+        self, targetDensity: float, tempGuessInC: float
     ) -> float:
-        """Get the temperature at which the perturbed density occurs."""
+        """Get the temperature at which the perturbed density occurs (in Celcius)."""
         # 0 at tempertature of targetDensity
         densFunc = lambda temp: self.density(Tc=temp) - targetDensity
         # is a numpy array if fsolve is called
-        tAtTargetDensity = float(fsolve(densFunc, temperatureGuessInC))
+        tAtTargetDensity = float(fsolve(densFunc, tempGuessInC))
         return tAtTargetDensity
 
     @property
     def liquidPorosity(self) -> float:
+        """Fraction of the material that is liquid void (unitless)."""
         return 0.0 if self.parent is None else self.parent.liquidPorosity
 
     @property
     def gasPorosity(self) -> float:
+        """Fraction of the material that is gas void (unitless)."""
         return 0.0 if self.parent is None else self.parent.gasPorosity
 
     def pseudoDensity(self, Tk: float = None, Tc: float = None) -> float:
         """
-        Return density that preserves mass when thermally expanded in 2D.
+        Return density that preserves mass when thermally expanded in 2D (in g/cm^3).
 
         Warning
         -------
-        This density will not agree with the component density since this method only expands in 2 dimensions.
-        The component has been manually expanded axially with the manually entered block hot height.
-        The density returned by this should be a factor of 1 + dLL higher than the density on the component.
-        density should be in agreement at both cold and hot temperatures as long as the block height is correct for
-        the specified temperature.
-        In the case of Fluids, density and density are the same as density is not driven by linear expansion, but
-        rather an explicit density function dependent on Temperature. linearExpansionPercent is zero for a fluid.
+        This will not typically agree with ``Material.density()`` or ``Component.density ()``
+        since this methods only expands in 2 dimensions. Depending on your use of
+        ``inputHeightsConsideredHot`` and ``Component.temperatureInC``, ``Material.psuedoDensity()``
+        may be a factor of (1+dLL) different than ``Material.density()`` or ``Component.density()``.
```

Comment:
Two minor typos:
1. ``Component.density()``
2. "....since this method only expands...."

Reply:
Fixed! Thanks!